backbone.stickit.js

```diff
@@ -174,7 +174,11 @@
 
             // Don't update the model if false is returned from the `updateModel` configuration.
             var currentVal = evaluateBoolean(config.updateModel, val, event, config);
-            if (currentVal) setAttr(model, modelAttr, val, options, config);
+            if (isPromise(currentVal)) {
+              currentVal.then(function(){
+                setAttr(model, modelAttr, val, options, config);
```

Comment:
I'd like to see some tests for this as I'm still having trouble picturing what you'd like the API to be. Mind adding a few test cases?